src/pages/documentation/resources/acp-end-of-support.md

```diff
@@ -0,0 +1,25 @@
+# ACP SDKs end-of-support
+
+## End of support for ACP iOS SDKs and Wrappers
```

Comment:
I don't think this H2 is necessary, since this is a one heading doc. (And this one doesn't say anything new).

Reply:
Ahh got it. Then yeah, that's fine.